pkg/kwokctl/pki/pki.go

```diff
@@ -106,7 +106,7 @@ func GenerateSignCert(cn string, caCert *x509.Certificate, caKey crypto.Signer,
 	certConfig := CertConfig{
 		CommonName:         cn,
 		Organization:       organizations,
-		Usages:             []x509.ExtKeyUsage{x509.ExtKeyUsageAny},
+		Usages:             []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth},
```

Comment:
```suggestion
		Usages:             []x509.ExtKeyUsage{
			x509.ExtKeyUsageAny,
			x509.ExtKeyUsageServerAuth,
		},
```

This should work.

but I'm not sure why ExtKeyUsageAny doesn't include ExtKeyUsageServerAuth, I've submitted an issue for this.
https://github.com/openssl/openssl/issues/23074

Reply:
I find that explicitly specify ExtKeyUsageServerAuth and ExtKeyUsageClientAuth should work.

i have updated the pr